cspell/.cspell.json

```diff
@@ -4,6 +4,7 @@
   "allowCompoundWords": false,
```

Comment:
Please confirm this warning.
![image](https://user-images.githubusercontent.com/31987104/177502976-e7195e1f-e71c-4c82-89fb-6a306311f3ac.png)